osu.Game/Screens/SelectV2/BeatmapLeaderboardScore.cs

```diff
@@ -697,52 +746,62 @@ private void load(OsuColour colours, OverlayColourProvider colourProvider)
 
         private partial class RankLabel : Container, IHasTooltip
         {
-            public RankLabel(int? rank, bool sheared)
+            private readonly bool darkText;
+            private readonly OsuSpriteText text;
+
+            public RankLabel(int? rank, bool sheared, bool darkText)
             {
+                this.darkText = darkText;
                 if (rank >= 1000)
                     TooltipText = $"#{rank:N0}";
 
-                Child = new OsuSpriteText
+                Child = text = new OsuSpriteText
                 {
                     Shear = sheared ? -OsuGame.SHEAR : Vector2.Zero,
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
-                    Font = OsuFont.GetFont(size: 20, weight: FontWeight.SemiBold, italics: true),
-                    Text = rank == null ? "-" : rank.Value.FormatRank().Insert(0, "#")
+                    Font = OsuFont.Style.Heading2,
+                    Text = rank == null ? "-" : rank.Value.FormatRank().Insert(0, "#"),
+                    Shadow = !darkText,
                 };
             }
 
+            [BackgroundDependencyLoader]
+            private void load(OverlayColourProvider colourProvider)
+            {
+                text.Colour = darkText ? colourProvider.Background3 : colourProvider.Content1;
+            }
+
             public LocalisableString TooltipText { get; }
         }
 
         private sealed partial class ColouredModSwitchTiny : ModSwitchTiny, IHasCustomTooltip<Mod>
         {
-            public Mod? TooltipContent { get; }
-
             [Resolved]
             private OverlayColourProvider colourProvider { get; set; } = null!;
 
             public ColouredModSwitchTiny(Mod mod)
                 : base(mod)
             {
-                TooltipContent = mod;
                 Active.Value = true;
             }
 
             public ITooltip<Mod> GetCustomTooltip() => new ModTooltip(colourProvider);
+
+            Mod? IHasCustomTooltip<Mod>.TooltipContent => (Mod)Mod;
         }
 
-        private sealed partial class MoreModSwitchTiny : CompositeDrawable
+        private sealed partial class MoreModSwitchTiny : CompositeDrawable, IHasPopover
```

Comment:
A button called a switch, with no sounds attached.

Reply:
Has been removed as part of moving from tiny mod icons to the old full ones.